src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,12 @@
+package lotto
+
+class LottoMachine {
+    fun draw(): LottoNumbers {
+        val result = mutableSetOf<LottoNumber>()
+
+        while (result.size < 6) {
+            result.add(LottoNumber())
+        }
+        return LottoNumbers(result.toMutableList())
```

Comment:
```suggestion
        return LottoNumbers(result.toList())
```
방어적 복사를 활용하려고 시도하신 모습 좋습니다. 👍 
하지만 MutableList로 전달을 하게 되면, 외부에서 값을 변경할 수 있습니다.
외부에서 변경하지 못하도록 개선해보면 어떨까요?